assets/src/js/tracker.js

```diff
@@ -171,6 +171,23 @@ function trackPageview() {
   window.setTimeout(() => { document.body.removeChild(i)}, 1000);
 }
 
+// add listener for history.pushState to detect SPA navigation
+var his = window.history
```

Comment:
`his`?

Reply:
Also I'm seeing arrow functions used in this file, which means you can use them and get rid of handling `this` yourselves.

Reply:
better to name it "she" 🥇

Reply:
i renamed it to `h`